lib/util/selection/index.js

```diff
@@ -289,10 +289,7 @@ function _selectAfterDelete(selPoint) {
         , selOffset = selPoint.offset;
 
     if (!selNode) return;
-    if (selNode.nodeType == Node.TEXT_NODE)
-        selNode.textContent = selNode.textContent.trimRight();
```

Comment:
trimRight was removing trailing whitespace which we require.

A cursory test for most text editing functionality doesn't flag up any new issues... QA informed to take their time with text editing tests to confirm

Reply:
UTF 🎆